tests/test_eigendistortion.py

```diff
@@ -193,16 +192,20 @@ def test_jac_vec_prod(self, state):
     def test_fisher_vec_prod(self, state):
         x, y, x_dim, y_dim, k = state
 
-        V = torch.randn((x_dim, k), device=DEVICE)
+        V, _ = torch.linalg.qr(torch.ones((x_dim, k), device=DEVICE), "reduced")
+        U = V.clone()
         Jv = autodiff.jacobian_vector_product(y, x, V)
         Fv = autodiff.vector_jacobian_product(y, x, Jv)
 
         jac = autodiff.jacobian(y, x)
 
-        Fv2 = jac.T @ jac @ V  # manually compute product to compare accuracy
+        Fv2 = jac.T @ jac @ U  # manually compute product to compare accuracy
 
         assert Fv.shape == (x_dim, k)
-        assert Fv2.allclose(Fv, rtol=1E-2)
+        if DEVICE != torch.device("cuda"):
+            assert Fv2.allclose(Fv, rtol=1E-2) 
+        else:
+            assert Fv2.allclose(Fv, atol=1E-6) 
```

Comment:
why check rtol for not cuda and atol for cuda? is there not a way to keep these the same just for clarity?

Reply:
made them both atol for clarity